modules/admin/proc-rhdh-deploy-aks-using-operator.adoc

```diff
@@ -85,6 +85,45 @@ kubectl -n <your_namespace> apply -f rhdh-ingress.yaml
 ----
 --
 
+. Create a ConfigMap named `app-config-rhdh` containing the {product-short} configuration using the following template:
++
+--
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: app-config-rhdh
+data:
+  "app-config-rhdh.yaml": |
+    app:
+      title: Red Hat Developer Hub
+      baseUrl: https://<rhdh_dns_name>
```

Comment:
Suggestion: Maybe we can add a line or two that explains what does rhdh_dns_name denote and from where the user can get this name?

Reply:
It's actually the app_address. I'm changing rhdh_dns_name to app_address.